event-sourcing/event-source/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>messaging-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>core</artifactId>
```

Comment:
Is the core and the messaging-core dependencies required?

Reply:
Yea, we now need the Enveloper.